modules/3d-tiles/src/lib/parsers/parse-3d-tile-point-cloud.js

```diff
@@ -158,7 +158,10 @@ async function parseDraco(tile, featureTable, batchTable, options, context) {
   let dracoBuffer;
   let dracoFeatureTableProperties;
   let dracoBatchTableProperties;
-  const batchTableDraco = batchTable && batchTable.getExtension('3DTILES_draco_point_compression');
+  const batchTableDraco =
```

Comment:
Did the existing line of code not work?

Reply:
It didn't work because there was no real batch table, only that extension in the Json. This seems to be how Cesium defines those attributes in point clouds. I took it from here: https://github.com/CesiumGS/cesium/blob/master/Source/Scene/PointCloud.js#L322